### PR TITLE
Testing | 30.0.4

### PR DESCRIPTION
**Pull request by ericgaspar**
- #729
- #753
- #758
- #746
- #763

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
(Dummy comment to test webhooks, please ignore)

**Comment by alexAubin** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -126,7 +124,7 @@ function list_installed_apps_not_compatible_with_future_version()
 current_version=$(grep OC_VersionString "$install_dir/version.php" | cut -d\' -f2)
 current_major_version=${current_version%%.*}
 
-last_version=$(ynh_read_manifest --manifest_key="resources.sources.main.url" | grep -o '[0-9][0-9]\.[0-9]\.[0-9][0-9]')
+last_version=$(ynh_read_manifest --manifest_key="resources.sources.main.url" | grep -o '[0-9][0-9]\.[0-9]\.[0-9]+')
```
Comment (alexAubin) [inline scripts/upgrade:127]:
```suggestion
last_version=$(ynh_read_manifest --manifest_key="resources.sources.main.url" | grep -o '[0-9][0-9]\.[0-9]\.[0-9]+')
```

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by Salamandar** [comment]
Worked flawlessly at home.

**Comment by Salamandar** [comment]
Worked flawlessly at home.

**Comment by ericgaspar** [comment]
!testme

File: tests.toml [hunk 2]
```diff
@@ -8,11 +8,11 @@ test_format = 1.0
     # Commits to test upgrade from
     # -------------------------------
 
-    test_upgrade_from.c5cf91ad.name = "Upgrade from 25.0.2"
-    test_upgrade_from.caf917f3.name = "Upgrade from 26.0.2"
+    #test_upgrade_from.c5cf91ad.name = "Upgrade from 25.0.2"
+    #test_upgrade_from.caf917f3.name = "Upgrade from 26.0.2"
```
Comment (alexAubin) [inline tests.toml:12]:
```suggestion
    test_upgrade_from.c5cf91ad.name = "Upgrade from 25.0.2"
    test_upgrade_from.caf917f3.name = "Upgrade from 26.0.2"
```

I'm thinking we should add these back now that CI is speedy again ?

**Comment by ericgaspar** [comment]
!testme

**Review by Tagadda — APPROVED** [review]
✨🚀

**Review by alexAubin — APPROVED** [review]

**Pull request merged**